client/src/components/Charts/Charts.jsx

```diff
@@ -85,13 +85,15 @@ class Charts extends React.Component {
       // time: this.props.time
       state: 'all', // default, overriden if state appears
       // published: false //default
+      lockItem: 'all',
+      lockType: this.props.lockType,
     };
-    this.props.filters.forEach((f) => params[f[0]] = f[1]);
-
+    this.props.filters.forEach((f) => params.lockItem = f[1]); // filters never has more than one, if dne foreach wont run tho
```

Comment:
I think I see what you mean. Instead of having separate 'state' and 'county' filters within filters, just have one pair of lockType and lockItem in filters and change the back end to simply this a bit. Implementing now

Reply:
Yeah, main problem is when filters[] does have more than one, like [[published, true], ['state', 'California']]